src/js/chartAuthoring.js

```diff
@@ -237,6 +347,24 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             $(this).find(dataEntryLabelSelector).val(currentData.label).trigger("change");
             $(this).find(dataEntryValueSelector).val(currentData.value).trigger("change");
         });
+
+        that.events.onUpdateDataEntryPanel.fire();
+    };
+
+    floe.chartAuthoring.addSonificationControlListeners = function(that) {
+        var playButton = that.locate("sonifierPlay"),
+            stopButton = that.locate("sonifierStop");
+
+        playButton.click(function(e) {
```

Comment:
These listeners form a pretty stereotypical pattern. I suggest drawing up some configuration accepted by the component and encoding this declaratively.